hack/deploy/copy_artifacts.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eo pipefail
+[[ -z "${BUILDDROP}" ]] && (echo "BUILDDROP is not set"; exit 1)
+[[ -z "${RELEASE_TAG}" ]] && (echo "RELEASE_TAG is not set"; exit 1)
+[[ -z "${VERSION}" ]] && (echo "VERSION is not set"; exit 1)
```

Comment:
what do these do?

Reply:
I just copy and pasted what we had elsewhere. @julienstroheker are they needed for tagging the artifacts during publish?